hypothesis-python/src/hypothesis/searchstrategy/types.py

```diff
@@ -24,7 +24,7 @@
 import fractions
 import functools
 import collections
-
+import numbers 
```

Comment:
This line should go just after `import decimal`; see [here](https://travis-ci.org/HypothesisWorks/hypothesis/jobs/443891792#L1087).  

You'll also need to remove any trailing whitespace, which also [fails the build](https://travis-ci.org/HypothesisWorks/hypothesis/jobs/443891790#L773).